### PR TITLE
feat: Add `InvoiceItems.lineType` field support

### PR DESCRIPTION
**Pull request by mbaleczny**

### CODE REVIEW

File: src/v0/invoices/README.md [hunk 1]
```diff
@@ -88,7 +89,8 @@ curl --location --request POST 'https://api.us.dev.vic.ai/v0/invoices' \
                     {
                         "externalId": "dep_9"
                     }
-                ]
+                ],
+                "lineType": "expense"
```
Comment (warmwaffles) [inline src/v0/invoices/README.md:93]:
```suggestion
                "lineType": "EXPENSE"
```

File: src/v0/invoices/README.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ curl --location --request POST 'https://api.us.dev.vic.ai/v0/invoices' \
                     {
                         "externalId": "prod_470"
                     }
-                ]
+                ],
+                "lineType": "item"
```
Comment (warmwaffles) [inline src/v0/invoices/README.md:73]:
```suggestion
                "lineType": "ITEM"
```

**Review by warmwaffles — APPROVED** [review]

File: vic.api.v0.yaml [hunk 3]
```diff
@@ -3855,6 +3860,11 @@ components:
       description: The type of invoice transaction.
       enum: [INVOICE, CREDITNOTE]
       default: INVOICE
+    InvoiceItemLineType:
+      type: string
+      description: The type of invoice item line.
+      enum: [ITEM, EXPENSE]
+      default: EXPENSE
```
Comment (warmwaffles) [inline vic.api.v0.yaml:3867]:
```suggestion
      enum: [item, expense]
      default: expense
```

Comment (mbaleczny) [reply vic.api.v0.yaml:3867]: I knew it! But I decided to go with the upper case, as I thought it should be like that for enums 😆

**Pull request merged**